src/api/app/models/package.rb

```diff
@@ -397,7 +397,12 @@ def unlock_by_request(request)
 
   def sources_changed(opts = {})
     dir_xml = opts[:dir_xml]
-    update_activity
+
+    # to call update_activity before filter
+    # NOTE: We need `Time.now`, otherwise the old tests suite doesn't work,
+    # remove it when removing the tests
+    update({updated_at: Time.now})
```

Comment:
In production when calling `update` the `updated_at` attribute is updated. But it is not updated in the tests, that is why we have to manually update it.

But this is not changing anything, this was already like that before.